code/modules/mob/living/silicon/ai/ai_defense.dm

```diff
@@ -69,6 +69,19 @@
 	to_chat(src, span_warning("[user] shorts out your access panel lock!"))
 	emagged = TRUE
 
+/mob/living/silicon/ai/wrench_act(mob/living/user, obj/item/tool)
+	. = ..()
+	if(user.combat_mode)
+		return
+	balloon_alert(user, "[!is_anchored ? "tightening" : "loosening"] bolts...")
+	balloon_alert(src, "bolts being [!is_anchored ? "tightened" : "loosened"]...")
+	if(!tool.use_tool(src, user, 4 SECONDS))
+		return TOOL_ACT_TOOLTYPE_SUCCESS
+	flip_anchored()
+	balloon_alert(user, "bolts [is_anchored ? "tightened" : "loosened"]")
+	balloon_alert(src, "bolts [is_anchored ? "tightened" : "loosened"]")
+	return TOOL_ACT_TOOLTYPE_SUCCESS
```

Comment:
Uh, why?

Reply:
@LemonInTheDark 
The AI deconstruction steps tell you to "bolt it down first!" if the AI is unanchored and you attempt deconning it. When I wrote those steps I assumed you already could wrench the AI like you can on baycode, it was only when I played on the live server and saw players hitting the AI with a wrench that I realized oh TG doesn't have that yet

Its not a balance issue, like I explained to Dorsi, if you can get up close to wrench the AI you can also just card/kill it. Also the AI can toggle itself anchored again with a verb anyway.